lib/elixir_tic_tac_toe.ex

```diff
@@ -1,6 +1,16 @@
+import GetUserMove
+import FirstComputerSpot
+import WinningCombo
+
 defmodule ElixirTicTacToe do
 
-  def printBoard(inputList) do
+  def playTicTacToe(inputList) do
```

Comment:
Also a little confused by this arg name. Seems like you use `inputList`, `possibleInputs`, and `board` interchangeably. The first two confuse me because they sound similar to what you have in get_user_move for figuring out which spaces are available.

Reply:
inputList has been changed to board. The input list was just another way of saying a passed through array, which in this case was the board.